lib/resolvers/place.js

```diff
@@ -23,6 +16,22 @@ module.exports = ({ models }) => ({ action, data = {}, id }, transaction) => {
     return Promise.resolve(null);
   };
 
+  const nacwoProfileId = data.nacwo; // old style tasks before mutliple nacwos per place were allowed
+  let requestedRoles = data.roles || [];
+
+  if (nacwoProfileId) {
+    const nacwoRoleId = await getNacwoRoleId(nacwoProfileId);
+    requestedRoles.push(nacwoRoleId);
+  }
```

Comment:
This is for backwards compatibility with in-flight tasks, right?

Reply:
correct